### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Rothen Evan, Thiébaud Jonathan

### DISCUSSION THREAD

**Discussion by JONATHAN-THIEBAUD** [heig-vd-dai-course/heig-vd-dai-course#550] [Show and tell]
### Description du projet
Nous souhaitons créer un système de gestion de jeux casino avec une interface CLI et une API. De manière général, le programme CLI permettra de gérer les jeux en CRUD et l'interface WEB permettra uniquement de filtrer les jeux et de les afficher dans des "cards" html. 

### API 
Notre API gérera les méthodes GET, POST, PATCH/PUT et DELETE

### Ressources 
Le système comportera deux ressources de données (entités) permettant de gérer les jeux. On retrouvera la table jeux qui contiendra toutes les informations pertinentes sur un jeux que l'on veut gérer, ainsi que la table catégorie sur laquelle la table jeux se réfère. 

**Contenu de jeux**
- id 
- name
- category (PK)
- rtp (return to player)
- volatility
- link (redirect to the real casino game)

**Contenu de description**
- id
- name

### Déploiement et infrastructure
- Inclure un déploiement de l'application avec docker et/ou docker compose
- Inclure un reverse proxy (Traefik) pour l'accessibilité https depuis une URL

Indiquez-nous si vous souhaitez obtenir plus de détails !

**Comment by ludelafo** [comment]
OK pour nous mais attention au temps : restez avec une API simple qui gère du JSON (pas besoin d'interface HTML) et utilisez curl plutôt que de développer votre propre CLI.

**Comment by JONATHAN-THIEBAUD** [comment]
## Rendu du projet 3 : 
Lien vers le repos git : https://github.com/JONATHAN-THIEBAUD/Casino
hash du dernier commit : d7119739561ceef3c6512f8138966934af4ee46e
